Utility/PltFileManager/PltFileManager.H

```diff
@@ -0,0 +1,46 @@
+#ifndef _PLTFILEMANAGER_H_
+#define _PLTFILEMANAGER_H_
+
+#include <AMReX_MultiFab.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_Geometry.H>
+
+namespace pele {
+namespace physics {
+
+class PltFileManager 
+{
+public:
+
+  void init(const std::string &a_pltFile);
```

Comment:
can we just have a constructor instead of an `init` ? Or is this because we are using it as a `static` in LM/C?